backends-velox/src/main/scala/org/apache/gluten/execution/ColumnarPartialProjectExec.scala

```diff
@@ -118,6 +118,9 @@ case class ColumnarPartialProjectExec(original: ProjectExec, child: SparkPlan)(
       case _: DecimalType => true
       case YearMonthIntervalType.DEFAULT => true
       case _: NullType => true
+      case _: ArrayType => true
+      case _: MapType => true
```

Comment:
If the element type of MapType is unsupported type, we should fallback. Please use function `doSchemaValidate` in class `VeloxValidatorApi`

Reply:
change it to `doSchemaValidate `